docs/backends.md

```diff
@@ -88,10 +99,58 @@ Content-Length: 129
 {"id":"ca219243-0dd4-4e6e-94ad-54fbf3dd8b32","name":"my dell server","description":"some description","address":"192.168.100.4"}
 ```
 
+## View Backend Information
+For a SEV enabled backend, the backend information can be viewed using prism. This information is measured by a rust script found [here](https://github.com/ultravioletrs/cocos/blob/main/scripts/backend_info/src/main.rs). Once compiled and the binary is stored in `/build`, the backend information can be measured by prism as shown below.
+
+On the backend page, click the Backend Information button:
+![backend_page](img/backend_page.png)
+
+If the measurement binary is absent and no measurement is found, an empty measurement file is displayed:
+
+![empty_info](img/empty_backend_info.png)
+
+If the measurement is present or measurement is found in the db, the measurement file will be available for download.
+
+![backend_info](img/backend_info.png)
+
+The backend_info.json file is useful in cocos for [attested TLS](https://docs.cocos.ultraviolet.rs/attestation/#attested-tls), and can be used to verify [attestation report](https://docs.cocos.ultraviolet.rs/cli/#fetch-and-validate-attestation-report). The file is provided to [cocos-cli](https://docs.cocos.ultraviolet.rs/cli/#backend-info) which can add measurement data or host data. The file contains the SnpPolicy and RootOfTrust as shown:
+
+```
+{
+  "snp_policy": {
+    "policy": 1966081,
+    "family_id": "AA==",
+    "image_id": "AA==",
+    "vmpl": {
+      "value": 0
+    },
+    "minimum_tcb": 1506397780360888800,
+    "minimum_launch_tcb": 1506397780360888800,
+    "require_author_key": false,
+    "measurement": "AA==",
+    "host_data": "AA==",
+    "report_id_ma": "AA==",
+    "chip_id": "GrFqQtRklrsjBslu9pcQ6X4rkftFW1Ar1oT+I4guQ1sVC6qakgSvEtE4P/SLSJ6mHNp0kY0mHnGpvz1Ov+k/w==",
+    "minimum_build": 7,
+    "minimum_version": "1.55",
+    "permit_provisional_firmware": false,
+    "require_id_block": false
+  },
+  "root_of_trust": {
+    "product": "Milan",
+    "check_crl": true,
+    "disallow_network": false
+  }
+}
+
+```
+
 ## Terminate Backend
 This is used to disconnect and close the associated backend connection. This is usually triggered when a certificate is revoked while the backend is connected using this certificate or user initiated for any reason.
 Please note that this action will stop all ongoing computations and stop manager and any running agent.
 
+Manual backend termination can not be done directly on ui, but can be done using the curl command below:
```

Comment:
...can not be done through the graphical user interface, but can be done using the HTTP API: